pkg/restapi/helper/request.go

```diff
@@ -40,8 +40,33 @@ type RevokeRequestInfo struct {
 	// Unique Suffix
 	DidUniqueSuffix string
 
+	// encoded one-time password for revoke operation
+	RecoveryOTP string
+}
+
+//RecoverRequestInfo is the information required to create recover request
+type RecoverRequestInfo struct {
+
+	// Unique Suffix of the did to be recovered
+	DidUniqueSuffix string
```

Comment:
I see its already labelled this way in previous commits, but note:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms